components/text-utils/text-utils.css

```diff
@@ -0,0 +1,50 @@
+h1,
+.xl-text {
+  font-size: 4.8rem;
+  line-height: 6.4rem;
+}
+h2,
+.large-text {
+  font-size: 3.2rem;
+  line-height: 4rem;
+}
+h3,
+.medium-text {
+  font-size: 2.4rem;
+  line-height: 3.2rem;
+}
+h4,
+.normal-text {
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+}
+h5,
```

Comment:
Just a suggestion, for readability, I like to keep one empty line between code blocks  :)  This applies to entire file.

Reply:
Fixed em. Thanks